docs/Contributing/pull-request.rst

```diff
@@ -1,3 +1,8 @@
+..
+    SPDX-License-Identifier: GPL-3.0-or-later
+    SPDX-FileCopyrightText: 2022 jwrober <jwrober at gmail.com>
+    SPDX-FileCopyrightText: 2022 panch93 <pranav.ppanch@gmail.com>
```

Comment:
So whats the norm for this ? We should decide on one I think. We use the github account or the lt.net account ? 
Here it is my lt.net account. Which is potentially confusing cuz my commits are linked to the email id, pranav.sampathkumar@gmail.com which is my github account.

Reply:
Anyways, My preference is Pranav Sampathkumar <pranav.sampathkumar@gmail.com>, cuz handles might change.